vizro-core/changelog.d/20240607_125345_huong_li_nguyen_create_kpi_cards.md

```diff
@@ -7,7 +7,7 @@ Uncomment the section that is right (remove the HTML comment wrapper).
 ### Highlights ✨
 
 - Introduce `Figure` as a new `Page` component, enabling all dash components to be reactive in
-  `Vizro`. See the [user guide on figure](XXXX) for more information. ([#493](https://github.com/mckinsey/vizro/pull/493))
+  `Vizro`. See the [user guide on figure](XXX) for more information. ([#493](https://github.com/mckinsey/vizro/pull/493), [#524](https://github.com/mckinsey/vizro/pull/524))
```

Comment:
Link will be updated to https://vizro.readthedocs.io/en/stable/pages/user-guides/figure/ after approval, otherwise the docs link won't get generated because it's failing the build as the link is not available yet